src/main.rs

```diff
@@ -109,15 +110,15 @@ enum CliCommand {
 
         /// Amount of DUSK to send
         #[clap(short, long)]
-        amt: f64,
+        amt: Dusk,
 
         /// Max amt of gas for this transaction
         #[clap(short = 'l', long)]
         gas_limit: Option<u64>,
 
-        /// Max price you're willing to pay for gas used (in LUX)
+        /// Max price you're willing to pay for gas used
```

Comment:
Tbh I don't like to express gasPrice in DUSK. I find it a bit uncomfortable (too many zeros)

Reply:
Solved in #16